src/vdaf/mastic.rs

```diff
@@ -580,7 +607,8 @@ where
         };
 
         let eval_proof = {
-            let mut eval_proof_xof = P::init(&[0; SEED_SIZE], &[]);
+            // TODO spec: Use a zero seed.
+            let mut eval_proof_xof = P::init(&[0; SEED_SIZE], &[&dst_usage(USAGE_EVAL_PROOF), ctx]);
```

Comment:
Or we could use an empty seed per the current spec. However that would require making `P` concrete (`XofTurboShake128`)